bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -360,7 +360,7 @@ mod tests {
             gas_left: 1337,
             output_data: Box::into_raw(Box::new([0xde, 0xad, 0xbe, 0xef])) as *const u8,
             output_size: 4,
-            release: None,
+            release: Some(release_stack_result),
```

Comment:
I don't think this is correct, as the output data here is just a boxed vector, while `release_stack_result` expects it was allocated via`std::alloc`.

I'd say you need to create a local release function in the tests which just do `Box::from_raw` to consume it.

Reply:
Both `release_stack_result` and `release_heap_result` call into `deallocate_output_data`, so there is no difference on that end.
It doesn't really expect that the buffer is allocated via `std::alloc` I think. As long as the pointer is coerced to a type which takes ownership of the memory, `Drop` can do the rest.

Reply:
There is no drop in `release_stack_result`, it uses `dealloc` directly.